shell/platform/darwin/ios/framework/Headers/FlutterPlugin.h

```diff
@@ -225,6 +225,42 @@ typedef void (*FlutterPluginRegistrantCallback)(NSObject<FlutterPluginRegistry>*
 - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result;
 @end
 
+#pragma mark -
+/***************************************************************************************************
+ * How the UIGestureRecognizers of a platform view should be blocked.
+ *
+ * UIGestureRecognizers of a platform views can be blocked based on the decisions made by the
+ * Flutter Framework. e.g. When an interact-able widget is covering the platform view and a gesture
+ * happened on the widget, The framework may decide to block the UIGestureRecognizers that are
+ * recognized (via the gesture recognized acton ) on the platform view if any.
+ *
+ * This policy describes how the blocking process is implemented.
```

Comment:
nit: This policy determines how Flutter blocks a platform view's UIGestureRecognizers.